client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import classNames from 'classnames';
+import { dateI18n } from '@wordpress/date';
+import { getHistory } from '@woocommerce/navigation';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, CardFooter, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import wcpayTracks from 'tracks';
+import type { Dispute } from 'wcpay/types/disputes';
+import { getAdminUrl } from 'wcpay/utils';
+import { formatExplicitCurrency } from 'wcpay/utils/currency';
+import { getDisputeFee, isInquiry } from 'wcpay/disputes/utils';
+import './style.scss';
+
+const formatUnixTimestamp = ( date: string, format: string ) =>
+	dateI18n( format, moment.unix( parseInt( date, 10 ) ).toISOString() );
+
+interface DisputeFooterProps {
+	dispute: Dispute;
+}
+
+const DisputeFooter: React.FC< DisputeFooterProps > = ( { dispute } ) => {
+	const isSubmitted = !! dispute.metadata.__evidence_submitted_at;
+	const isAccepted = dispute.metadata.__closed_by_merchant === '1';
+	const closedDateFormatted = dispute.metadata.__dispute_closed_at
+		? formatUnixTimestamp( dispute.metadata.__dispute_closed_at, 'M j, Y' )
+		: '-';
```

Comment:
What are the `-` fallback strings used for – would they ever get rendered?

Reply:
Understanding the flows, I'm 99% sure they won't.

However, I've left these for that 1% uncertainty since it is nicer to render `-` than `undefined` in the case of a mistake or a bug.